applications/GeoMechanicsApplication/tests/test_dirichlet_u.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsDirichletUTests(KratosUnittest.TestCase):
+    """
+    This class contains a test for displacement Dirichlet boundary condition
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def test_dirichlet_u(self):
+        """
+        4 element elongation test in 2 stages. The incremental elastic material should show 
+        linear increase in stress_yy over the steps, with continuity over the stages.
+        To reach this a saw tooth like table for the prescribed (stage) displacement is given.
+        H = 1 [m]
+        Young's modulus E = 30E+06 [N/m^2]
+        stage 1) elongate 0.05 [m] in 2 steps
+        stage 2) elongate another 0.05 [0.05] in 2 steps
+        """
+        test_name    = 'dirichlet_u'
+        project_path = test_helper.get_file_path(os.path.join('.', test_name))
+        n_stages     = 2
+        stages       = test_helper.get_stages(project_path,n_stages)
+
+        # name of output file
+        output_file_names = [os.path.join(project_path, 'dirichlet_u_stage' + str(i + 1) + '.post.res') for i in
```

Comment:
We can write the generation of the output file name in a more compact manner using Python's f-strings:
```suggestion
        output_file_names = [os.path.join(project_path, f'dirichlet_u_stage{i+1}.post.res') for i in
```

Reply:
Nicer, tried, it works.